storage/src/cache/fscache/mod.rs

```diff
@@ -257,4 +275,69 @@ impl FileCacheEntry {
             prefetch_config,
         })
     }
+
+    //restore index_map from blob file via seek_hole
+    #[cfg(target_os = "linux")]
```

Comment:
I wonder if we should disable fscahce on non-linux target OS altogether. fscache is surely a Linux thing.

Reply:
Yes, that should be better , but it seems hard to completely isolate the code for fscache and disable it. Any advises? @jiangliu

Reply:
It would be better to isolate fscache for Linux only. It's ok to move on with this PR and open another issue for conditionally enable fscache for linux only.